modules/get-started/pages/whats-new.adoc

```diff
@@ -29,3 +41,22 @@ This release includes several updates to xref:get-started:licensing/overview.ado
 
 - *Unified license management in Redpanda Console*: You can now upload and apply a single license key for both Redpanda Console and the connected Redpanda cluster through the Redpanda Console UI. Any existing license key is overridden by the new one.
 
+== New commands 
+
+The following `rpk` commands are new in this version:
+
+* xref:reference:rpk/rpk-cluster/rpk-cluster-storage-cancel-mount.adoc[`rpk cluster storage cancel-mount`]
+* xref:reference:rpk/rpk-cluster/rpk-cluster-storage-list-mount.adoc[`rpk cluster storage list-mount`]
```

Comment:
I think this may now be 'list-mountable'
@gene-redpanda to review this section.

Reply:
@micheleRP Correct, it's list-mountable. https://github.com/redpanda-data/docs/pull/725/files#diff-3e3155706ef086bcf367fd3a3f7be815b8800e7fcc2790997fd8be35327d957bR72

Reply:
@mattschumpert: Now we show both `rpk cluster storage list mount` and `rpk cluster storage list-mountable` [here](https://deploy-preview-811--redpanda-docs-preview.netlify.app/24.3/get-started/whats-new/#new-commands).